src/vs/platform/quickinput/browser/quickInputController.ts

```diff
@@ -140,6 +140,8 @@ export class QuickInputController extends Disposable {
 		const container = dom.append(this._container, $('.quick-input-widget.show-file-icons'));
 		container.tabIndex = -1;
 		container.style.display = 'none';
+		container.setAttribute('role', 'dialog');
+		container.setAttribute('aria-labelledby', 'quick-input-title');
```

Comment:
this is not so straightforward. The title is "Manage Trusted Extensions" in this case:
<img width="617" height="190" alt="Image" src="https://github.com/user-attachments/assets/270b034d-c751-452d-b2dc-138f302ebe66" />

but there is no title bar in this case:
<img width="621" height="456" alt="Image" src="https://github.com/user-attachments/assets/7c0091a9-203c-45c9-a361-4f7280f4ea60" />

Can you tell me why this container needs `the aria-labelledby`? The true aria label depends on a lot of different factors.